docs/change-media.md

```diff
@@ -0,0 +1,56 @@
+# Change Media
+
+The Player enables you to play several different media in the same instance of the player,
+All you have to do is call the `setMeida()` or `loadMeida()` API with the new `entryId / source`
+
+for example:
+
+```html
+<div id="{TARGET_ID}" style="width: 640px;height: 360px"></div>
+<script type="text/javascript" src="https://cdnapisec.kaltura.com/p/{PARTNER_ID}/embedPlaykitJs/uiconf_id/{UICONF_ID}"></script>
+<script type="text/javascript">
+  try {
+    var kalturaPlayer = KalturaPlayer.setup({
+      targetId: '{TARGET_ID}',
+      provider: {
+        partnerId: {PARTNER_ID},
+        uiConfId: {UICONF_ID}
+      },
+      playback: {
+        autoplay: true
+      }
+    });
+    kalturaPlayer.loadMedia({entryId: '{ENTRY_ID}'});
+
+    setTimeout((
+      kalturaPlayer.loadMedia({entryId: '{ANOTHER_ENTRY_ID}'});
+    ), 5000);
+
+  } catch (e) {
+    console.error(e.message);
+  }
+</script>
+```
+
+In that case, the player resets, and will play the newly provided media.
+
+The reset only includes everything related to the source / source metadata, **but keep the reset of the settings** which are not media specific, such as configuration / plugins
```

Comment:
reset includes only data structures which are related ...
**but keep the reset of the settings as is**